From b646787c313a85935a894d6fb50e60e30d9f3b33 Mon Sep 17 00:00:00 2001
From: Christopher Dykes <cdykes@fb.com>
Date: Mon, 22 Aug 2016 15:55:49 -0700
Subject: [PATCH] Fixup a reference to detail/FunctionalExcept.cpp

Summary: I missed this reference when I moved it to the portability folder. It is already being compiled at its new location.

Reviewed By: yfeldblum

Differential Revision: D3754227

fbshipit-source-id: 357b0c26ddbcefdc7640f6a334150abba90ed711
---
 folly/Makefile.am | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/folly/Makefile.am b/folly/Makefile.am
index 5df00b8..a51285f 100644
--- a/folly/Makefile.am
+++ b/folly/Makefile.am
@@ -497,10 +497,6 @@ if !HAVE_WEAK_SYMBOLS
 libfollybase_la_SOURCES += detail/MallocImpl.cpp
 endif
 
-if !HAVE_BITS_FUNCTEXCEPT_H
-libfollybase_la_SOURCES += detail/FunctionalExcept.cpp
-endif
-
 if HAVE_BOOST_CONTEXT
 nobase_follyinclude_HEADERS += \
 	fibers/AddTasks.h \
